### ISSUE TITLE
Styleguide not regenerating on update of README.md file

### ISSUE THREAD

**Issue by shanecarmody** [SC5/sc5-styleguide#1015]
Hi,

I was wondering if there was any way to get the styleguide to regenerate when I update the `README.md` (the file specified in the `overviewPath` option)? Each time I update this file I have to rerun the Gulp task to generate my styleguide. 

Thanks in advance.

**Comment by nategreen** [comment]
I think you can do this by telling Gulp to watch that file and run the styleguide task whenever it changes, the same way you do with your Sass/LESS files. So, if you copied the example task that looks like this:

```JavaScript
gulp.task('watch', ['styleguide'], function() {
  // Start watching changes and update styleguide whenever changes are detected
  // Styleguide automatically detects existing server instance
  gulp.watch(['*.scss'], ['styleguide']);
});
```

You'd want to add `'README.md'` to the first argument of `gulp.watch()`, giving you:

```JavaScript
gulp.task('watch', ['styleguide'], function() {
  // Start watching changes and update styleguide whenever changes are detected
  // Styleguide automatically detects existing server instance
  gulp.watch(['*.scss', 'README.md'], ['styleguide']);
});
```

**Comment by varya** [comment]
Thank you, @nategreen. That's correct.

**Comment by shanecarmody** [comment]
Maybe my issue title was badly worded. 

@nategreen your solution indeed regenerates the styleguide but does not refresh the content. So for example, when I add a paragraph to my `README.md` and save, I do not see the paragraph I added… I have to refresh manually. Do you know what I mean? 

Should I rather be using something like LiveReload?

**Comment by nategreen** [comment]
@shanecarmody Yeah, I think so.
